sprocs/assessment_groups_update.sql

```diff
@@ -13,7 +13,25 @@ DECLARE
     arg_user_id bigint;
     arg_group_id bigint;
     group_user text[];
+    default_role_id bigint;
 BEGIN
+    IF NOT EXISTS (
```

Comment:
Same comment as before: this smells funky to me. Let's see if we can do without this special value.

Reply:
See: https://github.com/PrairieLearn/PrairieLearn/pull/6109#discussion_r1042685844